flask/app/routes.py

```diff
@@ -277,10 +279,17 @@ def participants_list():
 @app.route('/api/analyses', methods=['GET'], endpoint='analyses_list')
 @login_required
 def analyses_list():
+    since_date = request.args.get('since', default='0001-01-01')
+    try:
+        since_date = datetime.strptime(since_date, "%Y-%m-%d").date()
```

Comment:
This works great! However, we want to query with more granularity than just the date (sorry if the requirement was unclear). Can the time be included as well?